sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -402,7 +402,7 @@ def test_partial_fit():
     assert_array_almost_equal(X_transform, transformer.transform(data))
 
     # check that if est doesn't have partial_fit, neither does SelectFromModel
-    transformer = SelectFromModel(estimator=RandomForestClassifier())
+    transformer = SelectFromModel(estimator=RandomForestRegressor())
```

Comment:
There's an error in CI related to this: 

```
sklearn/feature_selection/tests/test_from_model.py:405:45: F821 Undefined name `RandomForestRegressor`
```

Probably need to import the RFRegressor.